python/pyspark/sql/tests/test_arrow.py

```diff
@@ -487,10 +487,18 @@ def check_createDataFrame_respect_session_timezone(self, arrow_enabled):
 
     def test_createDataFrame_with_schema(self):
         pdf = self.create_pandas_data_frame()
-        df = self.spark.createDataFrame(pdf, schema=self.schema)
-        self.assertEqual(self.schema, df.schema)
-        pdf_arrow = df.toPandas()
-        assert_frame_equal(pdf_arrow, pdf)
+        for schema in [
+            self.schema,
+            (self.schema,),
```

Comment:
While I get the intention, let's probably don't do this. There are a lot of places like this, and I think it's not worthwhile fixing them all.

Reply:
Understood. Thanks for considering it.